src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -39,13 +96,33 @@ public PlayerEditor() {
 
         add(new JScrollPane(playerInfoList,
                 JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED, JScrollPane.HORIZONTAL_SCROLLBAR_NEVER), "span, grow");
+
+        playerInfoList.addListSelectionListener(event -> SwingUtilities.invokeLater(() -> {
```

Comment:
The event occurs on the swing thread, you DO NOT and SHOULD NOT use an invoke later for this, you want to do this synchronously when the selection changes